assets/app/index.html

```diff
@@ -135,6 +135,13 @@
     <!-- endbower -->
     <!-- endbuild -->
 
+
+    <!-- HACK!
+        Wiredeps isn't loading URI.js plugins when adding override to bower.json. need to figure out why
+    -->
+    <script src="bower_components/uri.js/src/URI.fragmentQuery.js"></script>
```

Comment:
@spadgett yup, your suggestion to use `URITemplate` works, I'm removing this dependency.